internal/resources/connections/resource_metrics_endpoint_scrape_job.go

```diff
@@ -80,6 +81,12 @@ func (r *resourceMetricsEndpointScrapeJob) Schema(ctx context.Context, req resou
 					stringplanmodifier.RequiresReplace(),
 				},
 			},
+			"enabled": schema.BoolAttribute{
```

Comment:
There is also the `disabled_reason` field, which I believe we can add as a computed field?

Reply:
Following up what we discussed:
We'll shelve this for later. Captured some thoughts here, please feel free to add to this if I missed anything https://github.com/grafana/cloud-onboarding/issues/7854